packages/blaze/util.js

```diff
@@ -0,0 +1,25 @@
+has = function (obj, key) {
```

Comment:
we shouldn't expose anything globally, use exports / imports instead.

maybe it would be better to use specific modules of lodash instead of reimplementing these functions here.